lib/procedures/ensure-manta-deployment-gz-links.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2019 Joyent, Inc.
+ */
+
+/*
+ * This is a Procedure to ensure the GZ has links to the manta-deployment
+ * tools in the manta-deployment zone, e.g. "/opt/smartdc/bin/manta-adm".
+ */
+
+'use strict';
+
+const assert = require('assert-plus');
+const fs = require('fs');
+const util = require('util');
+const vasync = require('vasync');
+const VError = require('verror');
+
+const errors = require('../errors');
+const Procedure = require('./procedure').Procedure;
+
+function EnsureMantaDeploymentGzLinksProcedure() {
+}
+util.inherits(EnsureMantaDeploymentGzLinksProcedure, Procedure);
+
+EnsureMantaDeploymentGzLinksProcedure.prototype.summarize =
+function summarize() {
+    return 'ensure /opt/smartdc/bin/manta-* tools are setup';
+};
+
+EnsureMantaDeploymentGzLinksProcedure.prototype.execute =
```

Comment:
compare to https://github.com/joyent/sdc-headnode/blob/3d07d324fdbd21536315356b47037a7a2d37f7c5/scripts/setup_manta_zone.sh#L178-L241